src/hooks/useUserByName.ts

```diff
@@ -0,0 +1,20 @@
+import { useGetUserByNameQuery } from '~gql';
+
+const useUserByName = (username: string) => {
```

Comment:
Same thought as above. From memory, your concern was that it just adds another layer of abstraction on top of codegen and isn't worth it given the only utility is destructuring the data.